### DISCUSSION TITLE
[HW2] Problem 1.2. Question about interpretation of negative inner product.

### DISCUSSION THREAD

**Discussion by HayeonLee** [seominjoon/kaist-ai605#124] [Q&A] [answered]
Does 'negative' inner product mean that the result of inner product calculation < 0?
If I understand correctly, by the definition of a metric that a metric is a non-negative real-valued function, it is already not a metric. Do we need further proof or an example for it or did I misunderstand the meaning of the negative inner product?

Thanks in advance!

**Comment by seominjoon** [comment, marked as answer]
Inner product is the higher the better, whereas regular metrics are the lower the better. "negative" is just to make both in the same direction.
